apps/fontKorean/metadata.json

```diff
@@ -0,0 +1,16 @@
+{ "id": "fontKorean",

```

Comment:
make the id all lower case as it is with e.g. `fontall`. Also for the e.g. folder name and others to make it similar to how `fontall` does it. Reference https://forum.espruino.com/comments/17353889/ .

Reply:
Thanks. I will make the change.